pwiz_tools/Skyline/TestUtil/TestFunctional.cs

```diff
@@ -1025,7 +1025,7 @@ public bool IsPauseForCoverShot
 
         public bool IsSavingCoverShots
         {
-            get { return /* CoverShotName != null*/ false; }
+            get { return IsPauseForCoverShot && CoverShotName != null; }
```

Comment:
I think this would only get used inside a PauseForCoverShot() call, right? Which should only happen when IsPauseForCoverShots is true.

Reply:
Fine to leave it this way, though.